com.unity.film-tv.toolbox/Tests/Editor/MaterialRemapperTest.cs

```diff
@@ -1,13 +1,37 @@
 using UnityEngine;
-using UnityEditor;
-using UnityEngine.TestTools;
 using NUnit.Framework;
-using System.Collections;
+using System.Linq;
+using UnityEditor.Formats.Alembic;
+using UnityEditor.SceneManagement;
 
-class EditorExampleTest {
+class EditorExampleTest
+{
+    MaterialRemapperModel m_Model;
 
     [Test]
-    public void EditorSampleTestSimplePasses() {
-        // Use the Assert class to test conditions.
+    public void TestMeshListCount() {
```

Comment:
Usually tests have descriptive names: eg: DuplicateMeshesAreConsideredOnlyOnce. Also NUnit's description attribute might be of interest.